src/main/osd/osd.c

```diff
@@ -404,6 +404,9 @@ void pgResetFn_osdConfig(osdConfig_t *osdConfig)
 
     osdConfig->canvas_cols = OSD_HD_COLS;
     osdConfig->canvas_rows = OSD_HD_ROWS;
+    
+    osdConfig->camera_fov_h = 138;
+    osdConfig->camera_fov_v = 75;
```

Comment:
Hardcoded values are not ideal here. Every camera can have different FOVs. If there is no other way, use macros, put them somewhere in the beginning of the file and give the hardcoded values names, for example `OSD_FOV_HORIZONTAL` and `OSD_FOV_VERTICAL`

Reply:
Yes that's clear I have not so much experience and can't get find out how to get this two vales from. Displayport

I post this question already in my future request but didn't do it here 😂
I need to get the rows and cols of the displayport in this function, but I do not know how I get it. Currently it´s hardcoded with 30 and 16. Maybe somebody can help me?
osdElementMovingHomePoint
//const uint8_t width = osdDisplayPort->cols;
//const uint8_t height = osdDisplayPort->rows;

Reply:
This I checked again, I have it in the config and did it similiar to `osdConfig()->rssi_alarm`, which also have some default values and the you can config it. I´m wrong?

Reply:
No, it's fine. I was thinking about something else but I guess it's not needed here.